src/apps/loan-list/list/loan-list.tsx

```diff
@@ -63,59 +56,44 @@ const LoanList: FC<LoanListProps> = ({ pageSize }) => {
   const [dueDate, setDueDate] = useState<string | null>(null);
   const [accepted, setAccepted] = useState<boolean>(false);
   const [modalDetailsId, setModalDetailsId] = useState<string | null>(null);
-  const [physicalLoans, setPhysicalLoans] = useState<LoanType[] | null>(null);
-  const [digitalLoans, setDigitalLoans] = useState<LoanType[] | null>(null);
-  const [physicalLoansDueDates, setPhysicalLoansDueDates] = useState<string[]>(
-    []
-  );
-  const { isSuccess, data } = useGetLoansV2();
-  const { data: publizonData } = useGetV1UserLoans();
+  const {
+    sortedByLoanDateFbs,
+    sortedByLoanDatePublizon,
+    stackedMaterialsDueDatesFbs
+  } = useLoans();
```

Comment:
I think it would be more nice with a return type that is something like: 
```typescript
type UseLoanProvider = {
  loansSortedByDate: Loan[];
  stackedMaterialsDueDates: Something[];
};

type UseLoans = {
  fbs: UseLoanProvider;
  publizon: UseLoanProvider;
};
```
Does it make sense?